tests/integration/test_pagure.py

```diff
@@ -295,6 +295,7 @@ def test_pr_info(self):
             pr_info.diff_url
             == "https://pagure.io/ogr-tests/pull-request/5#request_diff"
         )
+        assert isinstance(pr_info.head_commit, str)
```

Comment:
It does not differentiate the first/last commit.  (Or any other commit.) I would add the specific value as we do for all tests.